packages/common/package.json

```diff
@@ -28,7 +28,7 @@
     "./lib/loggers.js": "./lib/loggers-browser.js"
   },
   "scripts": {
-    "test": "NODE_OPTIONS=--experimental-vm-modules npx jest --silent --coverage --passWithNoTests --forceExit",
+    "test": "NODE_OPTIONS=--experimental-vm-modules npx jest --silent --coverage --passWithNoTests",
     "build": "npx tsc --project tsconfig.json",
```

Comment:
This PR adds a lot of calls to Ceramic core code into TestUtils in the `common` package.  Do we need to add devDependencies on the 'core' package?